### PR TITLE
Reinstate `get-balance` subcommand

### PR DESCRIPTION
**Pull request by georgepisaltu**
Fixes https://github.com/casper-network/casper-node/issues/3045

This PR reinstates the `get-balance` subcommand by removing the alias from `query-balance` and restoring the deleted functionality. Additionally, `get-balance` was also marked as deprecated in the `about` section of the command. I am open to any other suggestions towards marking the functionality as deprecated.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -101,6 +104,7 @@ fn cli() -> Command<'static> {
         .subcommand(SendDeploy::build(DisplayOrder::SendDeploy as usize))
         .subcommand(Transfer::build(DisplayOrder::Transfer as usize))
         .subcommand(MakeTransfer::build(DisplayOrder::MakeTransfer as usize))
+        .subcommand(GetBalance::build(DisplayOrder::GetBalance as usize))
```
Comment (rafal-ch) [inline src/main.rs:107]:
```suggestion
        .subcommand(GetBalance::build(DisplayOrder::GetBalance as usize).hide(true))
```

**Comment by rafal-ch** [comment]
Please also
* consider removing the now deprecated `get-balance` from the `README.md`, providing an alternative example that uses `query-balance`.
* update the `CHANGELOG.md` and mention that `get-balance` is reinstated but deprecated

**Comment by georgepisaltu** [comment]
> Please also
> 
>     * consider removing the now deprecated `get-balance` from the `README.md`, providing an alternative example that uses `query-balance`.
> 
>     * update the `CHANGELOG.md` and mention that `get-balance` is reinstated but deprecated

Done.

**Review by rafal-ch — APPROVED** [review]
LGTM :+1:

**Review by Fraser999 — APPROVED** [review]

**Pull request merged**